chunks/modelfreemodels.ipynb

```diff
@@ -395,11 +419,41 @@
     }
    },
    "source": [
-    "### Gaussian Processes\n",
+    "### Gaussian Processes in Data Analysis\n",
+    "\n",
+    "A draw from $P[y(x)|x^*]$ would represent a prior prediction for the function value $y(x^*)$\n",
```

Comment:
I don't understand the notation here. Surely the prior for the function value at x^* is P[y(x^\*)], and the posterior predictive distribution is P[y(x^\*)|{y(x_i)}].